apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -371,6 +371,10 @@ function writeFlagFile(packageInstallFolder: string): void {
   }
 }
 
+function getRushTempFolder(rushCommonFolder: string): string {
+  return process.env[RUSH_TEMP_FOLDER] || ensureAndJoinPath(rushCommonFolder, 'temp');
```

Comment:
We should also ensure the folder we get from the env variable exists. See the logic here: https://github.com/microsoft/web-build-tools/blob/master/apps/rush-lib/src/api/EnvironmentConfiguration.ts#L112

Reply:
Added a check to make sure the folder exists.